CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
-GitHub Issues is for reporting bugs in **<PODNAME>** and discussing features, bugs and general feedback. Be sure to check our [documentation](http://cocoadocs.org/docsets/<PODNAME>), [FAQ](https://github.com/hyperoslo/<PODNAME>/wiki/FAQ) and [past issues](https://github.com/hyperoslo/<PODNAME>/issues?state=closed) before opening any new issues.
+GitHub Issues is for reporting bugs in **<PODNAME>** and discussing features, bugs and general feedback. Be sure to check our [documentation](http://cocoadocs.org/docsets/<PODNAME>), [FAQ](https://github.com/<USERNAME>/<PODNAME>/wiki/FAQ) and [past issues](https://github.com/<USERNAME>/<PODNAME>/issues?state=closed) before opening any new issues.
```

Comment:
@sindrenm reads nicer but it doesn't work well when collaborating, people might forget do update this or remove that. I would do it for non-collaborative markdown documents.